robottelo/hosts.py

```diff
@@ -1392,15 +1394,25 @@ def is_upstream(self):
         :return: True if no downstream satellite RPMS are installed
         :rtype: bool
         """
-        return self.execute('rpm -q satellite-capsule &>/dev/null').status != 0
+        return self.execute(f'rpm -q {self.product_rpm_name}').status != 0
+
+    @cached_property
+    def is_stream(self):
+        """Check if the Capsule is a stream release or not
+
+        :return: True if the Capsule is a stream release
+        :rtype: bool
+        """
+        if self.is_upstream:
+            return False
+        return (
+            'stream' in self.execute(f'rpm -q --qf "%{{RELEASE}}" {self.product_rpm_name}').stdout
+        )
 
     @cached_property
     def version(self):
-        if not self.is_upstream:
-            version = self.execute('rpm -q satellite-capsule').stdout
-            return 'stream' if 'stream' in version else version.split('-')[2]
-        else:
-            return 'upstream'
+        rpm_name = self.upstream_rpm_name if self.is_upstream else self.product_rpm_name
+        return self.execute(f'rpm -q --qf "%{{VERSION}}" {rpm_name}').stdout
```

Comment:
```suggestion
        return self.execute(f'rpm -q --qf %{{VERSION}} {rpm_name}').stdout
```

Reply:
Better to be explicit than implicit. I mean, these quotes are not that much redundant. If we eventually decide to query `VERSION` and `RELEASE` in this command in the future, we would have to add quotes there anyway.

Reply:
Yes, these quotes really should be there